lib/generators/accessly/install/install_generator.rb

```diff
@@ -0,0 +1,51 @@
+require "rails/generators"
+require "rails/generators/active_record"
+
+module Accessly
+  module Generators
+    class InstallGenerator < Rails::Generators::Base
```

Comment:
I consulted Rails docs and the thoughtbot/clearance to define the migration generator